android/app/src/main/java/app/organicmaps/widget/placepage/PlaceDescriptionFragment.java

```diff
@@ -16,33 +19,161 @@
 
 import java.util.Objects;
 
-public class PlaceDescriptionFragment extends BaseMwmFragment
-{
-  public static final String EXTRA_DESCRIPTION = "description";
-  private static final String SOURCE_SUFFIX = "<p><b>wikipedia.org</b></p>";
-
-  @SuppressWarnings("NullableProblems")
-  @NonNull
-  private String mDescription;
-
-  @Override
-  public void onCreate(@Nullable Bundle savedInstanceState)
-  {
-    super.onCreate(savedInstanceState);
-    mDescription = Objects.requireNonNull(requireArguments()
-                                                 .getString(EXTRA_DESCRIPTION));
-  }
-
-  @Nullable
-  @Override
-  public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
-                           @Nullable Bundle savedInstanceState)
-  {
-    View root = inflater.inflate(R.layout.fragment_place_description, container, false);
-    WebView webView = root.findViewById(R.id.webview);
-    webView.loadData(mDescription + SOURCE_SUFFIX, Utils.TEXT_HTML, Utils.UTF_8);
-    webView.setVerticalScrollBarEnabled(true);
-    ViewCompat.setOnApplyWindowInsetsListener(root, WindowInsetUtils.PaddingInsetsListener.excludeTop());
-    return root;
-  }
-}
+public class PlaceDescriptionFragment extends BaseMwmFragment {
+    public static final String EXTRA_DESCRIPTION = "description";
+    private String getSourceSuffix() {
+        return "<p>" + getString(R.string.article_from_wikipedia) + "</p>";
```

Comment:
Would it be more convenient to insert a clickable link to the original article here?